websocket/frontend/src/todolist/TodoList.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import TodoForm from "./Todoform";
+import TodoItem from "./TodoItem";
+
+const apiURL = 'http://localhost:3001/api/v1/todos/';
+
+class TodoList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      items: []
+    };
+
+    this.updateTodoList = this.updateTodoList.bind(this);
+  };
+
+  componentDidMount() {
+    this.getTasks();
+  };
+
+  getTasks() {
+    fetch(apiURL)
+    .then(response => response.json())
+    .then(responseItems => {
+      this.setState({
+        items: responseItems.reverse()
+      });
+    });
+  };
+
+  updateTodoList(item) {
+    const currentItems = this.state.items;
+    this.setState({
+      items: [item, ...currentItems]
+    });
```

Comment:
```suggestion
    this.setState({
      items: [item, ...this.state.items]
    });
```
would this be better for you in terms of readability?